sdk/metrics/src/jmh/java/io/opentelemetry/sdk/metrics/internal/aggregator/HistogramValueGenerator.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.aggregator;
+
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.function.DoubleSupplier;
+
+/** Methods of generating values for histogram benchmarks. */
+@SuppressWarnings("ImmutableEnumChecker")
+public enum HistogramValueGenerator {
+  SAME_VALUE(() -> 100.0056),
```

Comment:
Curious if this value has any meaning, especially in terms of what the bucket is. Is it ok not to test various values?

Reply:
This was just to understand the cost of distributing over buckets vs. always placing in the same.

It's only useful if you're looking for where the slowdown is, not really as its own test.  I can remove, as this is NOT a use case to optimsie for.  The random value one is.